### PR TITLE
Set up sass importing, reset, and bourbon

### PR DESCRIPTION
**Pull request by creuter**
- Add sass-brunch to compile sass
- Add normalize, for a sane reset
- Add bourbon, for mixins
- Add bitters, for sane base styling
- Remove the default css file. Normalize can be imported separately, and bootstrap is bootstrap.

### CODE REVIEW

**Comment by creuter** [comment]
All of the .`scss` additions in the diff are from Bitters.

File: web/static/css/base/_base.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Bitters 1.2.0
+// http://bitters.bourbon.io
+// Copyright 2013-2015 thoughtbot, inc.
+// MIT License
+
+@import "variables";
+
+// Neat Settings -- uncomment if using Neat -- must be imported before Neat
```
Comment (drapergeek) [inline web/static/css/base/_base.scss]: Should this stay in?

File: web/static/css/base/_base.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Bitters 1.2.0
```
Comment (drapergeek) [inline web/static/css/base/_base.scss]: Do we normally leave this in?

Comment (creuter) [inline web/static/css/base/_base.scss]: Yes, but there's no great reason to.

**Comment by drapergeek** [comment]
LGTM

**Pull request merged**